modules/manage/pages/iceberg/iceberg-topics-aws-glue.adoc

```diff
@@ -0,0 +1,213 @@
+= Query Iceberg Topics using AWS Glue
+:description: Add Redpanda topics as Iceberg tables that you can query from AWS Glue Data Catalog.
+:page-categories: Iceberg, Tiered Storage, Management, High Availability, Data Replication, Integration
+:page-beta: true
+ifdef::env-cloud[]
+:rpk-install-doc: manage:rpk/rpk-install.adoc
+endif::[]
+ifndef::env-cloud[]
+:rpk-install-doc: get-started:rpk-install.adoc
+endif::[]
+
+
+[NOTE]
+====
+include::shared:partial$enterprise-license.adoc[]
+====
+
+// tag::single-source[]
+
+This guide walks you through querying Redpanda topics as Iceberg tables stored in AWS S3, using a catalog integration with https://docs.aws.amazon.com/glue/latest/dg/components-overview.html#data-catalog-intro[AWS Glue^]. For general information about Iceberg catalog integrations in Redpanda, see xref:manage:iceberg/use-iceberg-catalogs.adoc[].
+
+== Prerequisites
+
+* Redpanda version 25.1.7 or later.
+* xref:{rpk-install-doc}[`rpk`] installed or updated to the latest version.
+ifdef::env-cloud[]
+** You can also use the Redpanda Cloud API to xref:manage:cluster-maintenance/config-cluster.adoc#set-cluster-configuration-properties[reference secrets in your cluster configuration].
+endif::[]
+ifndef::env-cloud[]
+* xref:manage:tiered-storage.adoc#configure-object-storage[Object storage configured] for your cluster and xref:manage:tiered-storage.adoc#enable-tiered-storage[Tiered Storage enabled] for the topics for which you want to generate Iceberg tables.
++
+You also use the S3 bucket URI to set the base location for Glue Data Catalog.
+endif::[]
+* Admin permissions to create IAM policies and roles in AWS.
+
+== Limitations
+
+=== Nested partition spec support
+
+Glue does not support partitioning on nested fields. If Redpanda detects that
```

Comment:
```suggestion
AWS Glue does not support partitioning on nested fields. If Redpanda detects that
```

Reply:
I looked through their user guide and they always use **AWS Glue**. Let's do the same.